agent/engine/docker_task_engine.go

```diff
@@ -290,33 +290,39 @@ func (engine *DockerTaskEngine) synchronizeContainerStatus(container *api.Docker
 		} else {
 			container.DockerID = describedContainer.ID
 			container.Container.SetKnownStatus(dockerStateToState(describedContainer.State))
+			container.Container.SetCreatedAt(describedContainer.Created)
+			container.Container.SetStartedAt(describedContainer.State.StartedAt)
```

Comment:
Is there any possibility `describedContainer.State` and `describedContainer.Config` is nil?

Reply:
I don't think so